server/service/src/invoice/prescription/update/mod.rs

```diff
@@ -352,7 +357,7 @@ mod test {
                     diagnosis_id: _,
                 } = get_update();
                 u.name_link_id = patient_id.unwrap();
-                u.clinician_link_id = clinician_id;
+                u.clinician_link_id = clinician_id.unwrap().value;
```

Comment:
```suggestion
                u.clinician_link_id = clinician_id.map(|c| c.value).unwrap_or_default();
```
This works, but feels a bit clunky, I think what you have is totally fine.

Slightly better is this, but feels weird having two functions, and the second one is the one that runs on a `Some` value?
```suggestion
               u.clinician_link_id =  clinician_id.map_or_else(|| None, |c| c.value);
```

The problem with just use the map function is that it automatically wraps whatever you're mapping in Some()
So it doesn't let you just return a Some yourself. THis is why we were getting an extra `Some` in the type definition...

Reply:
Thanks for the tips :)